index.js

```diff
@@ -101,6 +101,12 @@ var defaultCommands = {
     invariantSet(spec);
     return value;
   },
+  $toggle: function(target, nextObject, spec) {
+    invariantToggle(target, nextObject);
+    var nextObjectCopy = copy(nextObject);
+    nextObjectCopy[target] = !nextObject[target];
```

Comment:
Would it make more sense to have target be an array so that multiple values can be toggled?

```js
update({a: true, b: false}, {$toggle: ['a', 'b']}) // {a: false, b: true}
```

The implementation can be as follows:

```js
invariant(
   Array.isArray(value),
  'update(): expected spec of $toggle to be an array; got %s. ' +
  'Did you forget to wrap the key(s) in an array?',
  value
);
nextObjectCopy = targets.length ? copy(nextObject) : nextObject;
targets.forEach(function(target) {
  nextObjectCopy[target] = !nextObject[target];
})
```

Reply:
Absolutely! I think it's a great idea. Thank you – I'll update the PR soon